specification/src/main/asciidoc/coreprofile/CoreProfileDefinition.adoc

```diff
@@ -17,14 +17,14 @@ The following technologies are required components of the Core Profile:
 
 === Optional Components
 
-There are no optional components in the Core Profile.
+The optional Jakarta XML Binding support in Jakarta RESTful Web Services implementations is not required for the Core Profile.
+
+Jakarta Expression Language support is not required in  Jakarta Contexts and Dependency Injection Lite implementations since this is part of the Full CDI specification.
+
+The Java SE section of the CDI 4.0 are not required for Core Profile implementations. This is because that specification states that CDI implementations that support the Java SE API are required to support the Full version of the CDI specification.
```

Comment:
Missing `specification` word I think, replace `The Java SE section of the CDI 4.0 are not required for Core Profile implementations` with `The Java SE section of the CDI 4.0 specification is not required for Core Profile implementations`.

Also could  `This is because that specification states that CDI implementations that support the Java SE API are required to support the Full version of the CDI specification.` be changed to something like `Only Full CDI implementations are required to support the (CDI) Java SE API classes.`  That might help users to grok the reference to `Java SE` more easily.

Reply:
Ok, updated